netlogo-gui/src/main/app/DirtyMonitor.scala

```diff
@@ -38,8 +38,8 @@ with SaveModel.Controller
   private var loading = true
   private var _modelDirty = false
   private var priorTempFile = Option.empty[Path]
-  private var _codeWindow : Option[javax.swing.JDialog] = None
-  def setCodeWindow(codeWindow: Option[javax.swing.JDialog]): Unit = {
+  private var _codeWindow : Option[javax.swing.JFrame] = None
+  def setCodeWindow(codeWindow: Option[javax.swing.JFrame]): Unit = {
```

Comment:
I think `JFrame` is already imported in this file so we can drop the `javax.swing` part of the package name.